.github/workflows/alter-primary-key-false-test.yml

```diff
@@ -0,0 +1,48 @@
+name: alter-primary-key-false-test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        distribution: [ 'adopt' ]
+    name: Java ${{ matrix.distribution }} sample
+    steps:
+
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: ${{ matrix.distribution }}
+          cache: maven
+
+      - name: add host and copy properties
+        run: |
+          echo -e "127.0.0.1   pd \n127.0.0.1   tikv" | sudo tee -a /etc/hosts
+          sudo cp -r config /config
+          echo "spark.sql.catalog.tidb_catalog=org.apache.spark.sql.catalyst.catalog.TiCatalog" > tidb_config.properties
+          mv tidb_config.properties core/src/test/resources/tidb_config.properties
+
+      - name: build tidb cluster
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+          cd config
+          ~/.tiup/bin/tiup playground ^4 --kv.config tikv.toml --pd.config pd.toml --db.config  tidb-alter-primary-key-false.toml --tiflash=0 --without-monitor
```

Comment:
> 1. What is ^4 stand for?
> 2. We can use [`sed`](https://unix.stackexchange.com/questions/159367/using-sed-to-find-and-replace) command to replace the config `alter-primary-key` rather than duplicate a new config file

^4 means use tidb4.0.
OK,I change to use sed.